Changes to the credit card operation

Shaggs opened the pull request:
I've removed the card number and auth number. Storing credit card details in this manor I'm sure is not safe?

This also opens this up for the use of a manual EFTPOS machine. Some of these machines allow the operator to enter the amounts manually as apposed to integrating it.

Integrating it would be the next major change but lack of access to this machine make it hard for us  to code it.

The changes now will bring up a message box asking if the EFTPOS machine approved the transaction. If yes is pressed then the sale will go though. If no is selected the the current transaction is cancelled. I did try to make it go back to the sale screen but was unable to find it.

I also added myself as a contributor as I've now made quite a few changes. Also updated the copyright to update from 2015 to 2019 (I'm sure this was a slight oversight)

As normal @hiramvillarreal its for you to approve

Shaggs committed:
add crdit card changes

hiramvillarreal commented:
I've removed the card number and auth number. Storing credit card details in this manor I'm sure is not safe?
It is just for reference you should keep the printed voucher it is juts to identified what printed voucher was used to paid what transaction please do not remove.
Integrating it would be the next major change but lack of access to this machine make it hard for us to code it

Yes that will be the next as option but since then we need to keep those fields.


The changes now will bring up a message box asking if the EFTPOS machine approved the transaction. If yes is pressed then the sale will go though. If no is selected the the current transaction is cancelled. I did try to make it go back to the sale screen but was unable to find it.
When we had knowledge how the EFTPOS works probably we will find it gives the auth number to be filled automatically by the terminal.

We could add to configuration whet the EFTPOS machine is available that could make the field read only for cashier and a manual setting when there not the EFTPOS machine avalible




De: Shane Rees [mailto:notifications@github.com]
Enviado el: martes, 29 de enero de 2019 07:36 a. m.
Para: hiramvillarreal/iotpos
CC: Hiram Villarreal; Mention
Asunto: [hiramvillarreal/iotpos] Changes to the credit card operation (#26)


I've removed the card number and auth number. Storing credit card details in this manor I'm sure is not safe?

This also opens this up for the use of a manual EFTPOS machine. Some of these machines allow the operator to enter the amounts manually as apposed to integrating it.

Integrating it would be the next major change but lack of access to this machine make it hard for us to code it.

The changes now will bring up a message box asking if the EFTPOS machine approved the transaction. If yes is pressed then the sale will go though. If no is selected the the current transaction is cancelled. I did try to make it go back to the sale screen but was unable to find it.

I also added myself as a contributor as I've now made quite a few changes. Also updated the copyright to update from 2015 to 2019 (I'm sure this was a slight oversight)

As normal @hiramvillarreal<https://github.com/hiramvillarreal> its for you to approve

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/hiramvillarreal/iotpos/pull/26

Commit Summary

  *   add crdit card changes

File Changes

  *   M src/iotposview.cpp<https://github.com/hiramvillarreal/iotpos/pull/26/files#diff-0> (71)
  *   M src/ui/mainview.ui<https://github.com/hiramvillarreal/iotpos/pull/26/files#diff-1> (61)

Patch Links:

  *   https://github.com/hiramvillarreal/iotpos/pull/26.patch
  *   https://github.com/hiramvillarreal/iotpos/pull/26.diff

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/hiramvillarreal/iotpos/pull/26>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AKFJWEFCw5X35_uXzDW-94-pqaWM1A56ks5vIFxCgaJpZM4aYLbI>.

hiramvillarreal commented:
I also added myself as a contributor as I've now made quite a few changes. Also updated the copyright to update from 2015 to 2019 (I'm sure this was a slight oversight)
Yes please add you name as contributor as forget to tellpu since the first day thanks buddy for your help.



De: Shane Rees [mailto:notifications@github.com]
Enviado el: martes, 29 de enero de 2019 07:36 a. m.
Para: hiramvillarreal/iotpos
CC: Hiram Villarreal; Mention
Asunto: [hiramvillarreal/iotpos] Changes to the credit card operation (#26)


I've removed the card number and auth number. Storing credit card details in this manor I'm sure is not safe?

This also opens this up for the use of a manual EFTPOS machine. Some of these machines allow the operator to enter the amounts manually as apposed to integrating it.

Integrating it would be the next major change but lack of access to this machine make it hard for us to code it.

The changes now will bring up a message box asking if the EFTPOS machine approved the transaction. If yes is pressed then the sale will go though. If no is selected the the current transaction is cancelled. I did try to make it go back to the sale screen but was unable to find it.

I also added myself as a contributor as I've now made quite a few changes. Also updated the copyright to update from 2015 to 2019 (I'm sure this was a slight oversight)

As normal @hiramvillarreal<https://github.com/hiramvillarreal> its for you to approve

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/hiramvillarreal/iotpos/pull/26

Commit Summary

  *   add crdit card changes

File Changes

  *   M src/iotposview.cpp<https://github.com/hiramvillarreal/iotpos/pull/26/files#diff-0> (71)
  *   M src/ui/mainview.ui<https://github.com/hiramvillarreal/iotpos/pull/26/files#diff-1> (61)

Patch Links:

  *   https://github.com/hiramvillarreal/iotpos/pull/26.patch
  *   https://github.com/hiramvillarreal/iotpos/pull/26.diff

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/hiramvillarreal/iotpos/pull/26>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AKFJWEFCw5X35_uXzDW-94-pqaWM1A56ks5vIFxCgaJpZM4aYLbI>.

hiramvillarreal commented:
It is just for reference you should keep the printed voucher it is juts to identified what printed voucher was used to paid what transaction please do not remove those field since they are already and there is a DB fild those store that fields.

Until we lear how the EFTPOS works probably we will find the way how  it gives the auth number to be filled automatically by the com and make the fields read only .

hiramvillarreal commented:
Yes Plase add you Name an others updates to credits.

Shaggs committed:
minor fix

Shaggs committed:
typo

Shaggs committed:
change from mysql to mariadb

With the new raspbian (buster) mysql is no longer supported.
This fix now installs mariadb instead

Shaggs committed:
Merge pull request #1 from Shaggs/mariadb-patch

change from mysql to mariadb
